supportbot/supportbot/app.py

```diff
@@ -23,7 +23,7 @@ def run_app(config):
 
     app = App(token=os.environ.get("SLACK_BOT_TOKEN"))
 
-    database_client_cached = DatabaseClient(os.environ.get("SPREADSHEET_ID"))
+    database_client_cached = MeshDBDatabaseClient(os.environ.get("MESHDB_AUTH_TOKEN"))
```

Comment:
Is this actually cached?  The old client pulled all data on initialization.  Maybe rename otherwise?

Reply:
Not cached. It pulls at request time. Renamed